pages/login.py

```diff
@@ -121,18 +121,21 @@ def submit_consent(username_input):
                 sheet_condition = st.session_state.condition.split(' ', 1)[1]
                 print(f'sheet condition {sheet_condition}')
                 st.session_state['sheet'] = exponential_backoff(client.open, sheet_condition)  
-                st.session_state['user_worksheet'] = exponential_backoff(st.session_state['sheet'].worksheet, st.session_state.username)  
-                st.session_state['demographics'] = exponential_backoff(st.session_state['sheet'].worksheet, 'Demographics')  
-                all_values = exponential_backoff(st.session_state['user_worksheet'].get, 'H2:H50')
-                print(f'all values: {all_values}')
+                st.session_state['user_worksheet'] = exponential_backoff(st.session_state['sheet'].worksheet, st.session_state.username)
+                st.session_state['demographics'] = exponential_backoff(st.session_state['sheet'].worksheet, 'Demographics')
+                all_values = exponential_backoff(st.session_state['user_worksheet'].get, 'H2:H37')
+                print(f'all values: {all_values}, len: {len(all_values)}')
                 if not all_values or (len(all_values) == 1 and not all_values[0]):
                     st.session_state.questions_done = -1
-                elif len(all_values) >= 37:
+                    st.session_state.page =   "demographics" # begin_tutorial or instruction?
+                elif len(all_values) == 36:
```

Comment:
could there be any cases where len(all_values) are > 36? Should we consider using a less strict condition like len(all_values) >= 36 instead?

Reply:
that could work but since we are only grabbing values from row 2-37 and we have a single submit then we would have at most 36 rows. can include though.